src/alphabets/dna.rs

```diff
@@ -34,7 +34,7 @@ pub fn n_alphabet() -> Alphabet {
 
 /// The IUPAC DNA alphabet (uppercase and lowercase).
 pub fn iupac_alphabet() -> Alphabet {
-    Alphabet::new(b"ACGTURYSWKMBDHVNacgturyswkmbdhvn")
+    Alphabet::new(b"ACGTRYSWKMBDHVNZacgtryswkmbdhvnz")
```

Comment:
Adding Z is fine, but can you keep U (see [here](https://en.wikipedia.org/wiki/Nucleic_acid_notation))?